app/controllers/contacts_controller.rb

```diff
@@ -99,6 +87,27 @@ def contact
     end
   end
 
+  private def changes_requested_humanized(changes_requested)
+    changes_requested.map do |change|
+      case change[:field]
+      when :country_iso2
+        change_from = Country.find_by(iso2: change[:from]).name_in(:en)
+        change_to = Country.find_by(iso2: change[:to]).name_in(:en)
```

Comment:
Now you're still doing the same thing twice. What about a function that takes "field name, and some value (no matter whether it's the old or the new value" to do the complicated logic? You can then _call_ this helper twice, once for the old and once for the new.

In other words, the "heavy" logic of converting a country iso code to its English name should only be coded once. You are free to call this logic twice, but coding it twice is not very clean 😅

Reply:
Done